src/database/scripts/startDockerPostgres.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+# Copyright 2020-2021 OCAD University
+#
+# Licensed under the New BSD license. You may not use this file except in
+# compliance with this License.  You may obtain a copy of the License at
+# https://github.com/fluid-project/preferencesServer/blob/main/LICENSE
+
+# Starts up a Postgres database using a docker image.  If it is not present, it
+# is downloaded (pulled) from dockerhub.
+
+# Default values
+POSTGRES_MAIN_CONTAINER=${POSTGRES_MAIN_CONTAINER:-"postgresdb"}
+PGPORT=${PGPORT:-5432}
+POSTGRES_USER=${PGUSER:-"admin"}
+PGDATABASE=${PGDATABASE:-"fluid_prefsdb"}
+POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-"asecretpassword"}
+POSTGRES_IMAGE=${POSTGRES_IMAGE:-"postgres:13.1-alpine"}
+
+log() {
+  echo "$(date +'%Y-%m-%d %H:%M:%S') - $1"
+}
+
+run() {
```

Comment:
This definition of a `run()` shell function is not used anywhere.  It can be removed.